bench-vortex/src/engines/ddb/mod.rs

```diff
@@ -157,7 +157,7 @@ impl DuckDBCtx {
             }
             BenchmarkDataset::ClickBench { single_file, .. } => {
                 let file_glob = if *single_file {
-                    format!("{base_dir}hits.{extension}")
+                    format!("{base_dir}hits_0.{extension}")
```

Comment:
We should really kill the single_file version

Reply:
I guess that's a good sign as any that no one uses it, I'll add that to this PR